tests/components/zwave_js/test_api.py

```diff
@@ -1,12 +1,14 @@
 """Test the Z-Wave JS Websocket API."""
+from unittest.mock import patch
+
 from zwave_js_server.event import Event
 
+from homeassistant.components.zwave_js.api import ENTRY_ID, ID, NODE_ID, TYPE
 from homeassistant.components.zwave_js.const import DOMAIN
-from homeassistant.components.zwave_js.websocket_api import ENTRY_ID, ID, NODE_ID, TYPE
 from homeassistant.helpers.device_registry import async_get_registry
 
 
-async def test_websocket_api(hass, integration, multisensor_6, hass_ws_client):
+async def test_api(hass, integration, multisensor_6, hass_ws_client):
```

Comment:
This is just testing the websocket api.

Reply:
Right. The function itself is tested in the library.

Reply:
```suggestion
async def test_websocket_api(hass, integration, multisensor_6, hass_ws_client):
```